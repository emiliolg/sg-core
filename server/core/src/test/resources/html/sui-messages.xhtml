<html sui-view="tekgenesis.service.html.Views" sui-messages="quotes : tekgenesis.samples.service.FamousQuotes">
    <head>
    </head>
    <body>
    </body>
</html>
