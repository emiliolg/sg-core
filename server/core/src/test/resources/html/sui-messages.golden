<!DOCTYPE html>
<html sui-scope="/html/sui-messages.xhtml/0">
    <head>
    </head>
    <body>
    
<script class="suiScopesCache">
//<![CDATA[
var suiScopesCache = {"/html/sui-messages.xhtml/0":{"quotes":{"JOBS":"Stay hungry, stay foolish.","PLATO":"Life must be lived as play.","DALAI":"Be kind whenever possible. It is always possible."}}};
//]]>
</script>
</body>
</html>